README.md

```diff
@@ -28,4 +28,26 @@ Within the repo you'll find the following directories and files:
 │   ├── js
 │   ├── fonts
 │   └── templates
-├── docs
\ No newline at end of file
+├── docs
+```
+
+### So you want to build the less on your own? 
+
+Nothing easier than that! Simply include our styes.less files like: 
+`@import "bower_components/hyTechne/src/less/styles";`.
+
+Remember to fix the paths for fonts by overwriting the path variables 
+in a less include after the style.less include like: 
+
+`@import "_resource-paths";`
+
+and add in the file something like: 
+
+```
+// Use this file to point to the right folders
+// for correct paths for icon-fonts etc 
+
+@bootstrap-path: '../../bower_components/bootstrap'; // Path to bootstrap folder
+@bower-path: '../../bower_components';  // Path to bower_components folder
+@hytech-path: '../../bower_components/hyTechne'; // Path to hytech folder
```

Comment:
this should be achievable by using the bower-path variable with our fixed hyTechne folder name as bower will always install it under the registered folder name, so I'm not sure that we really need to have the extra variable at this level.